# Исследование надежности заемщиков


Во второй части проекта вы выполните шаги 3 и 4. Их вручную проверит ревьюер.
Чтобы вам не пришлось писать код заново для шагов 1 и 2, мы добавили авторские решения в ячейки с кодом. 



## Откройте таблицу и изучите общую информацию о данных

**Задание 1. Импортируйте библиотеку pandas. Считайте данные из csv-файла в датафрейм и сохраните в переменную `data`. Путь к файлу:**

`/datasets/data.csv`

In [1]:
import warnings
warnings.filterwarnings('ignore')

In [2]:
import pandas as pd

try:
    data = pd.read_csv('/datasets/data.csv')
except:
    data = pd.read_csv('https://code.s3.yandex.net/datasets/data.csv')

**Задание 2. Выведите первые 20 строчек датафрейма `data` на экран.**

In [3]:
data.head(20)

children  days_employed  dob_years            education  education_id  \
0          1   -8437.673028         42               высшее             0   
1          1   -4024.803754         36              среднее             1   
2          0   -5623.422610         33              Среднее             1   
3          3   -4124.747207         32              среднее             1   
4          0  340266.072047         53              среднее             1   
5          0    -926.185831         27               высшее             0   
6          0   -2879.202052         43               высшее             0   
7          0    -152.779569         50              СРЕДНЕЕ             1   
8          2   -6929.865299         35               ВЫСШЕЕ             0   
9          0   -2188.756445         41              среднее             1   
10         2   -4171.483647         36               высшее             0   
11         0    -792.701887         40              среднее             1   
12         0            NaN         65              среднее             1   
13         0   -1846.641941         54  неоконченное высшее             2   
14         0   -1844.956182         56               высшее             0   
15         1    -972.364419         26              среднее             1   
16         0   -1719.934226         35              среднее             1   
17         0   -2369.999720         33               высшее             0   
18         0  400281.136913         53              среднее             1   
19         0  -10038.818549         48              СРЕДНЕЕ             1   

       family_status  family_status_id gender income_type  debt  \
0    женат / замужем                 0      F   сотрудник     0   
1    женат / замужем                 0      F   сотрудник     0   
2    женат / замужем                 0      M   сотрудник     0   
3    женат / замужем                 0      M   сотрудник     0   
4   гражданский брак                 1      F   пенсионер     0   
5   гражданский брак                 1      M   компаньон     0   
6    женат / замужем                 0      F   компаньон     0   
7    женат / замужем                 0      M   сотрудник     0   
8   гражданский брак                 1      F   сотрудник     0   
9    женат / замужем                 0      M   сотрудник     0   
10   женат / замужем                 0      M   компаньон     0   
11   женат / замужем                 0      F   сотрудник     0   
12  гражданский брак                 1      M   пенсионер     0   
13   женат / замужем                 0      F   сотрудник     0   
14  гражданский брак                 1      F   компаньон     1   
15   женат / замужем                 0      F   сотрудник     0   
16   женат / замужем                 0      F   сотрудник     0   
17  гражданский брак                 1      M   сотрудник     0   
18    вдовец / вдова                 2      F   пенсионер     0   
19         в разводе                 3      F   сотрудник     0   

     total_income                                 purpose  
0   253875.639453                           покупка жилья  
1   112080.014102                 приобретение автомобиля  
2   145885.952297                           покупка жилья  
3   267628.550329              дополнительное образование  
4   158616.077870                         сыграть свадьбу  
5   255763.565419                           покупка жилья  
6   240525.971920                       операции с жильем  
7   135823.934197                             образование  
8    95856.832424                   на проведение свадьбы  
9   144425.938277                 покупка жилья для семьи  
10  113943.491460                    покупка недвижимости  
11   77069.234271       покупка коммерческой недвижимости  
12            NaN                         сыграть свадьбу  
13  130458.228857                 приобретение автомобиля  
14  165127.911772              покупка жилой недвижимости  
15  116820.904450 

**Задание 3. Выведите основную информацию о датафрейме с помощью метода `info()`.**

In [4]:
data.info()

<class 'pandas.core.frame.DataFrame'>
RangeIndex: 21525 entries, 0 to 21524
Data columns (total 12 columns):
 #   Column            Non-Null Count  Dtype  
---  ------            --------------  -----  
 0   children          21525 non-null  int64  
 1   days_employed     19351 non-null  float64
 2   dob_years         21525 non-null  int64  
 3   education         21525 non-null  object 
 4   education_id      21525 non-null  int64  
 5   family_status     21525 non-null  object 
 6   family_status_id  21525 non-null  int64  
 7   gender            21525 non-null  object 
 8   income_type       21525 non-null  object 
 9   debt              21525 non-null  int64  
 10  total_income      19351 non-null  float64
 11  purpose           21525 non-null  object 
dtypes: float64(2), int64(5), object(5)
memory usage: 2.0+ MB


## Предобработка данных

### Удаление пропусков

**Задание 4. Выведите количество пропущенных значений для каждого столбца. Используйте комбинацию двух методов.**

In [5]:
data.isna().sum()

children               0
days_employed       2174
dob_years              0
education              0
education_id           0
family_status          0
family_status_id       0
gender                 0
income_type            0
debt                   0
total_income        2174
purpose                0
dtype: int64

**Задание 5. В двух столбцах есть пропущенные значения. Один из них — `days_employed`. Пропуски в этом столбце вы обработаете на следующем этапе. Другой столбец с пропущенными значениями — `total_income` — хранит данные о доходах. На сумму дохода сильнее всего влияет тип занятости, поэтому заполнить пропуски в этом столбце нужно медианным значением по каждому типу из столбца `income_type`. Например, у человека с типом занятости `сотрудник` пропуск в столбце `total_income` должен быть заполнен медианным доходом среди всех записей с тем же типом.**

In [6]:
for t in data['income_type'].unique():
    data.loc[(data['income_type'] == t) & (data['total_income'].isna()), 'total_income'] = \
    data.loc[(data['income_type'] == t), 'total_income'].median()

### Обработка аномальных значений

**Задание 6. В данных могут встречаться артефакты (аномалии) — значения, которые не отражают действительность и появились по какой-то ошибке. таким артефактом будет отрицательное количество дней трудового стажа в столбце `days_employed`. Для реальных данных это нормально. Обработайте значения в этом столбце: замените все отрицательные значения положительными с помощью метода `abs()`.**

In [7]:
data['days_employed'] = data['days_employed'].abs()

**Задание 7. Для каждого типа занятости выведите медианное значение трудового стажа `days_employed` в днях.**

In [8]:
data.groupby('income_type')['days_employed'].agg('median')

income_type
безработный        366413.652744
в декрете            3296.759962
госслужащий          2689.368353
компаньон            1547.382223
пенсионер          365213.306266
предприниматель       520.848083
сотрудник            1574.202821
студент               578.751554
Name: days_employed, dtype: float64

У двух типов (безработные и пенсионеры) получатся аномально большие значения. Исправить такие значения сложно, поэтому оставьте их как есть. Тем более этот столбец не понадобится вам для исследования.

**Задание 8. Выведите перечень уникальных значений столбца `children`.**

In [9]:
data['children'].unique()

array([ 1,  0,  3,  2, -1,  4, 20,  5])

**Задание 9. В столбце `children` есть два аномальных значения. Удалите строки, в которых встречаются такие аномальные значения из датафрейма `data`.**

In [10]:
data = data[(data['children'] != -1) & (data['children'] != 20)]

**Задание 10. Ещё раз выведите перечень уникальных значений столбца `children`, чтобы убедиться, что артефакты удалены.**

In [11]:
data['children'].unique()

array([1, 0, 3, 2, 4, 5])

### Удаление пропусков (продолжение)

**Задание 11. Заполните пропуски в столбце `days_employed` медианными значениями по каждого типа занятости `income_type`.**

In [12]:
for t in data['income_type'].unique():
    data.loc[(data['income_type'] == t) & (data['days_employed'].isna()), 'days_employed'] = \
    data.loc[(data['income_type'] == t), 'days_employed'].median()

**Задание 12. Убедитесь, что все пропуски заполнены. Проверьте себя и ещё раз выведите количество пропущенных значений для каждого столбца с помощью двух методов.**

In [13]:
data.isna().sum()

children            0
days_employed       0
dob_years           0
education           0
education_id        0
family_status       0
family_status_id    0
gender              0
income_type         0
debt                0
total_income        0
purpose             0
dtype: int64

### Изменение типов данных

**Задание 13. Замените вещественный тип данных в столбце `total_income` на целочисленный с помощью метода `astype()`.**

In [14]:
data['total_income'] = data['total_income'].astype(int)

### Обработка дубликатов

**Задание 14. Обработайте неявные дубликаты в столбце `education`. В этом столбце есть одни и те же значения, но записанные по-разному: с использованием заглавных и строчных букв. Приведите их к нижнему регистру. Проверьте остальные столбцы.**

In [15]:
data['education'] = data['education'].str.lower()

**Задание 15. Выведите на экран количество строк-дубликатов в данных. Если такие строки присутствуют, удалите их.**

In [16]:
data.duplicated().sum()

71

In [17]:
data = data.drop_duplicates()

### Категоризация данных

**Задание 16. На основании диапазонов, указанных ниже, создайте в датафрейме `data` столбец `total_income_category` с категориями:**

- 0–30000 — `'E'`;
- 30001–50000 — `'D'`;
- 50001–200000 — `'C'`;
- 200001–1000000 — `'B'`;
- 1000001 и выше — `'A'`.


**Например, кредитополучателю с доходом 25000 нужно назначить категорию `'E'`, а клиенту, получающему 235000, — `'B'`. Используйте собственную функцию с именем `categorize_income()` и метод `apply()`.**

In [18]:
def categorize_income(income):
    try:
        if 0 <= income <= 30000:
            return 'E (низкий доход)'
        elif 30001 <= income <= 50000:
            return 'D (средний доход)'
        elif 50001 <= income <= 200000:
            return 'C (высокий доход)'
        elif 200001 <= income <= 1000000:
            return 'B (очень высокий доход)'
        elif income >= 1000001:
            return 'A (сверхдоход)'
    except:
        pass

In [19]:
data['total_income_category'] = data['total_income'].apply(categorize_income)

**Задание 17. Выведите на экран перечень уникальных целей взятия кредита из столбца `purpose`.**

In [20]:
data['purpose'].unique()

array(['покупка жилья', 'приобретение автомобиля',
       'дополнительное образование', 'сыграть свадьбу',
       'операции с жильем', 'образование', 'на проведение свадьбы',
       'покупка жилья для семьи', 'покупка недвижимости',
       'покупка коммерческой недвижимости', 'покупка жилой недвижимости',
       'строительство собственной недвижимости', 'недвижимость',
       'строительство недвижимости', 'на покупку подержанного автомобиля',
       'на покупку своего автомобиля',
       'операции с коммерческой недвижимостью',
       'строительство жилой недвижимости', 'жилье',
       'операции со своей недвижимостью', 'автомобили',
       'заняться образованием', 'сделка с подержанным автомобилем',
       'получение образования', 'автомобиль', 'свадьба',
       'получение дополнительного образования', 'покупка своего жилья',
       'операции с недвижимостью', 'получение высшего образования',
       'свой автомобиль', 'сделка с автомобилем',
       'профильное образование', 'высшее об

**Задание 18. Создайте функцию, которая на основании данных из столбца `purpose` сформирует новый столбец `purpose_category`, в который войдут следующие категории:**

- `'операции с автомобилем'`,
- `'операции с недвижимостью'`,
- `'проведение свадьбы'`,
- `'получение образования'`.

**Например, если в столбце `purpose` находится подстрока `'на покупку автомобиля'`, то в столбце `purpose_category` должна появиться строка `'операции с автомобилем'`.**

**Используйте собственную функцию с именем `categorize_purpose()` и метод `apply()`. Изучите данные в столбце `purpose` и определите, какие подстроки помогут вам правильно определить категорию.**

In [21]:
def categorize_purpose(row):
    try:
        if 'автом' in row:
            return 'операции с автомобилем'
        elif 'жил' in row or 'недвиж' in row:
            return 'операции с недвижимостью'
        elif 'свад' in row:
            return 'проведение свадьбы'
        elif 'образов' in row:
            return 'получение образования'
    except:
        return 'нет категории'

In [22]:
data['purpose_category'] = data['purpose'].apply(categorize_purpose)

In [23]:
data.head()

children  days_employed  dob_years education  education_id  \
0         1    8437.673028         42    высшее             0   
1         1    4024.803754         36   среднее             1   
2         0    5623.422610         33   среднее             1   
3         3    4124.747207         32   среднее             1   
4         0  340266.072047         53   среднее             1   

      family_status  family_status_id gender income_type  debt  total_income  \
0   женат / замужем                 0      F   сотрудник     0        253875   
1   женат / замужем                 0      F   сотрудник     0        112080   
2   женат / замужем                 0      M   сотрудник     0        145885   
3   женат / замужем                 0      M   сотрудник     0        267628   
4  гражданский брак                 1      F   пенсионер     0        158616   

                      purpose    total_income_category  \
0               покупка жилья  B (очень высокий доход)   
1     приобретение автомобиля        C (высокий доход)   
2               покупка жилья        C (высокий доход)   
3  дополнительное образование  B (очень высокий доход)   
4             сыграть свадьбу        C (высокий доход)   

           purpose_category  
0  операции с недвижимостью  
1    операции с автомобилем  
2  операции с недвижимостью  
3     получение образования  
4        проведение свадьбы

### Шаг 3. Исследуйте данные и ответьте на вопросы

#### 3.1 Есть ли зависимость между количеством детей и возвратом кредита в срок?

In [24]:
children_debt = data.groupby('children')['debt'].agg(['count', 'sum']) #рассчитаем по столбцу дети количество строк и сумму относительно просрочки
children_debt['result_children'] = (children_debt['sum'] / children_debt['count'])
display(children_debt.sort_values('result_children'))

count   sum  result_children
children                              
5             9     0         0.000000
0         14091  1063         0.075438
3           330    27         0.081818
1          4808   444         0.092346
2          2052   194         0.094542
4            41     4         0.097561

In [25]:
data.groupby('children')['debt'].mean()

children
0    0.075438
1    0.092346
2    0.094542
3    0.081818
4    0.097561
5    0.000000
Name: debt, dtype: float64

**Вывод:** 
- Прямой зависимости по просрочке кредита от колличества детей нет. 
- заемщики с 1, 2 и 4 детьми имеют примерно одинаковый показатель просрочки
- менее допускают просрочку бездетные заемщики
- низкий показатель просрочки по заемщикам с 5 детьми обусловлен малым количеством заемщиков на уровне погрешности.

#### 3.2 Есть ли зависимость между семейным положением и возвратом кредита в срок?

In [26]:
family_status_debt = data.groupby('family_status')['debt'].agg(['count', 'sum'])
family_status_debt['result_family_status'] = (family_status_debt['sum'] / family_status_debt['count'])
display(family_status_debt.sort_values('result_family_status'))

# Ваш код будет здесь. Вы можете создавать новые ячейки.

count  sum  result_family_status
family_status                                          
вдовец / вдова           951   63              0.066246
в разводе               1189   84              0.070648
женат / замужем        12261  927              0.075606
гражданский брак        4134  385              0.093130
Не женат / не замужем   2796  273              0.097639

In [27]:
data.groupby('family_status')['debt'].mean()

family_status
Не женат / не замужем    0.097639
в разводе                0.070648
вдовец / вдова           0.066246
гражданский брак         0.093130
женат / замужем          0.075606
Name: debt, dtype: float64

**Вывод:** 
- самый большой процент задолженности по возврату кредитов у "не женат/не замужем" и "гражданский брак"
- наиболее дисциплинированы вдовцы и в разводе
- в совокупности заемщики, в браке или бывшие в браке, более дисциплинированные заемщики.


#### 3.3 Есть ли зависимость между уровнем дохода и возвратом кредита в срок?

In [28]:
total_income_debt = data.groupby('total_income_category')['debt'].agg(['count', 'sum'])
total_income_debt['result_total_income'] = (total_income_debt['sum'] / total_income_debt['count'])
display(total_income_debt.sort_values('result_total_income'))# Ваш код будет здесь. Вы можете создавать новые ячейки.

count   sum  result_total_income
total_income_category                                    
D (средний доход)          349    21             0.060172
B (очень высокий доход)   5014   354             0.070602
A (сверхдоход)              25     2             0.080000
C (высокий доход)        15921  1353             0.084982
E (низкий доход)            22     2             0.090909

**Вывод:** 
- прямой корелляции между доходом и задолженностью по возврату кредитов 
- средний и очень высокий доход - низкий процент просрочки
- высокий доход - средний процент просрочки (возможно указан завышенный доход)
- выборка клиентов со сверхдоходами мало репрезентативна
- заемщики с низким доходом показали самый большой % задолженности по возврату кредитов, но видимо в силу низкой одобряемости - подборка тоже не репрезентативна

#### 3.4 Как разные цели кредита влияют на его возврат в срок?

In [29]:
purpose_debt = data.groupby('purpose_category')['debt'].agg(['count', 'sum'])
purpose_debt['result_purpose_category'] = (purpose_debt['sum'] / purpose_debt['count'])
display(purpose_debt.sort_values('result_purpose_category'))

# Ваш код будет здесь. Вы можете создавать новые ячейки.

count  sum  result_purpose_category
purpose_category                                             
операции с недвижимостью  10751  780                 0.072551
проведение свадьбы         2313  183                 0.079118
получение образования      3988  369                 0.092528
операции с автомобилем     4279  400                 0.093480

**Вывод:** 
- наиболее показательные зависимости задолженностью по возврату кредитов от целей кредитов.
- Самые дисциплинированные заемщики на Недвижимость и свадьбу.
- получение образования и приобретение автомобиля - наиболее рискованные кредиты.
- если в случае просрочки по кредиту за автомобиль банк подстрахован, то на образование дефолтность выше.

#### 3.5 Как разные социальные статусы влияют на возврат кредита в срок?

In [30]:
income_type_debt = data.groupby('income_type')['debt'].agg(['count', 'sum'])
income_type_debt['result_income_type'] = (income_type_debt['sum'] / income_type_debt['count'])
display(income_type_debt.sort_values('result_income_type'))

count   sum  result_income_type
income_type                                     
предприниматель      2     0            0.000000
студент              1     0            0.000000
пенсионер         3812   216            0.056663
госслужащий       1451    86            0.059269
компаньон         5047   374            0.074103
сотрудник        11015  1054            0.095688
безработный          2     1            0.500000
в декрете            1     1            1.000000

###### Добавил еще один срез для анализа.
Срез по социальному статусу - очень показательный.
Самые дисциплинированные заемщки Пенсионеры и Госслужащие.

#### 3.5 Приведите возможные причины появления пропусков в исходных данных.

*Ответ:* 
- человеческий фактор - ошибки ввода данных, это и минусы в стаже и количестве детей. Вероятно 20 детей тоже ошибка связанная с написанием или вводом. Я бы ее заменил на 2, но шаблон не предусматривал это.
- Отсутствие данных по стажу работы и ежемесячному доходу у некоторых пенсионеров - вероятно не является у них обязательным для заполнения полем.
- сокрытие информации заемщиком или ошибочные сведения - неразборчивый почерк, не понял что писать в анкете,
- фрод или мошенничество
- техническая - например соединение баз из двух и более. 


#### 3.6 Объясните, почему заполнить пропуски медианным значением — лучшее решение для количественных переменных.

*Ответ:* 
С помощью медианы можно получить более точные данные и правильнее интерпретировать статистику. Среднее значение из-за разброса крайних точек может дать искаженное значение для статистики.

### Шаг 4: общий вывод.

Основные критерии для оценки заемщика на основании данных о просрочках заемщиков: семейное положение, уровень дохода и цель кредита.
Количество детей не явно влияет на дисциплину заемщиков.
Дисциплинированные заемщики: 
- в браке или был в браке, 
- средний доход 30-50 т.р.
- цель кредита - недвижимость и свадьба

Менее дисциплинированные заемщики:
- не в браке
- указан высокий доход
- цель - образование или автомобиль

Рекомендации для банка:
- увеличить контроль за качеством заполения скоринговой анкеты
- дополнительное обеспечение по кредитам на автомобиль и образование
- приоритет для заемщиков в браке или бывшие в браке
- высокий доход не гарантирует обязательности заемщика - усилить контроль за данными по доходу

## Расширенный вывод:
Исследование зависимости качества заемщиков от основных критериев.
### Цель:

###### Определить, какие факторы влияют на качество заемщика и снижение дефолтности клиента.

#### Основные данные в базе данных:
— количество детей в семье
— общий трудовой стаж в днях
— возраст клиента в годах
— уровень образования клиента
— идентификатор уровня образования
— семейное положение
— идентификатор семейного положения
— пол клиента
— тип занятости
— имел ли задолженность по возврату кредитов
— ежемесячный доход
— цель получения кредита

#### Задача исследования - исследовать гипотезы:

- Есть ли зависимость между количеством детей и возвратом кредита в срок?
- Есть ли зависимость между семейным положением и возвратом кредита в срок?
- Есть ли зависимость между уровнем дохода и возвратом кредита в срок?
- Как разные цели кредита влияют на его возврат в срок?

### Выводы:

##### Есть ли зависимость между количеством детей и возвратом кредита в срок?

Данная гипотеза имеет частичное подтверждение в данных - **бездетные заемщики имеют меньше просрочек по возврату кредитов- 7,54%** при средней просрочке **8,44%**. Колличество детей в семье **не имеет большого влияния** на дефолтность заемщика.
1 ребенок 9,23%
2 ребенка 9,45%
3 ребенка 8,18% (низкая выборка)
4 ребенка 9,76% (низкая выборка)
Таким образом для скоринг должен учитывать только наличие или отсутствие детей. 

##### Есть ли зависимость между семейным положением и возвратом кредита в срок?

Данная гипотеза подтвердилась в большей степени. В результате анализа выявлено, что **заемщики не в браке или, живущие в гражданском браке** менее дисциплинированы в отношении кредитов, чем **заемщики в браке или ранее состоявшие в нем**. Вероятно ответственность заемщиков состоящих(состоявших) в браке, в независимости от того двое супругов или один - значительно ниже средней просрочки **8,44%**.
- Не женат / не замужем    9,76%
- гражданский брак         9,31%

- женат / замужем          7,56%
- в разводе                7,06%
- вдовец / вдова           6,62%

Рекомендации по данному параметру при скоринге - приорит заемщикам в браке или состоявшим в нем ранее.

##### Есть ли зависимость между уровнем дохода и возвратом кредита в срок?

Уровень дохода влияет на качество заемщиков в разрезе - **высокий/очень высокий** доход.
Заемщики с низким доходом допускают самое большое колличесво просрочки. При этом самый низкий уровень просрочки у следующей градации - средний доход от 30 000 до 50 000 рублей.
D (средний доход)	    6,02%
B (очень высокий доход)	7,06%
A (сверхдоход)	        8,00%
C (высокий доход)	    8,50%
E (низкий доход)	    9,09%

По колличеству данных в базе данных стоит отметить низкую релевантность выборки по заемщикам со сверхвысоким доходам - *22*, и низким доходам - *25*. Вероятно одобрение кредитов по экстремумам шкалы дохлодов низкое.
Заемщиков с средним доходом - *349*, что тоже не очень высокая выборка, и хоть дефолтность у них самая низкая, вряд ли можно делать вывод об увеличении кредитования данной категории.
Основная масса кредитов в категориях **высокий доход** и **очень высокий доход** и там диапазон просрочки от 7,06% до 8,5% - в пределах среднего значения.
Рекомендации для скорринга: сохранить существющую систему скоринга.

##### Есть ли зависимость между целью кредита и его возвратом в срок?

Мы выделили 4 основные цели кредита:
- операции с недвижимостью	10751	7,26%
- проведение свадьбы	     2313	7,91%
- получение образования	     3988	9,25%
- операции с автомобилем	 4279	9,35%
Самая низкодефолтная цель кредита - недвижимость, беспокойство о наличии жилья и ответстовенность перед близкими стимулируют своевременные оплаты. Вторая по качество цель - Свадьба. Невысокие суммы кредита и, как ранее отвмечали, заемщики в браке более дисциплинированы.
Обе цели кредита имеют процент просрочек ниже среднего значения **8,44%**.
Две другие цели кредита - образование и автомобиль - значительно превосходят по количеству просрочек средний показатель.

##### Как разные социальные статусы влияют на возвратом кредита в срок?

Выборка по таким статсам, как Предприниматель, студент, безработный, в декрете - нерепрезентативна

- предприниматель	2
- студент	        1	
- безработный	    2
- в декрете	    1

Среди остальных заемщиков самыу дисциплинированные пенсионеры и госслужащие. Пенсионеры привыкли распределять доходы и расходы и рассчитывать их. Госслужащие - стабиольность дохода и репутация.
- пенсионер	 5,67%
- госслужащий	5,93%
- компаньон	 7,41%
- сотрудник	9,57%
В то же время самая массовая категория заемщиков - **сотрудник (11 015)** имеет самый высокий процент дефолта.


### Рекомендации:

- для скоринга использовать зависимости - Цель кредита, Семейное положение
- бездетные заемщики более своевременно выплачивают кредит
- наемные работники (сотрудники) самая дефолтная категория, рекомендовано поручительство работодателя.
- данные по уровню дохода и количеству детей требуют дополнительно исследования с большей выборкой.
